integrations/yocto.rst

```diff
@@ -0,0 +1,296 @@
+
+    def deploy(self):
+        # Deploy everything from this eclipse/mosquitto package
+        copy(self, "*", src=self.package_folder, dst=self.deploy_folder)
+
+Another option is using the :ref:`deployers<reference_extensions_deployers>`,
+ which will copy all artifacts, including package dependencies to your installation folder.
+
+
+Setting up a Yocto SDK
+----------------------
+
+Yocto SDKs are completely self-contained, there is no dependency on libraries of the build machine or tools installed in it. The SDK is a
+cross-building toolchain matching the target and it is generated from that specific configuration. This means that you will have to use a
+different SDK toolchain to build for a different target architecture or that some SDK's may have specific settings to enable some system
+dependency of the final target and those libraries will be available in the SDK.
+
+You can `create your own Yocto SDKs <https://docs.yoctoproject.org/sdk-manual/appendix-obtain.html#building-an-sdk-installer>`_
+or download and use `the prebuilt ones <http://downloads.yoctoproject.org/releases/yocto/yocto-5.1.2/toolchain/x86_64/>`_.
+
+**In the case that you are using CMake** to create the Conan packages, Yocto injects a toolchain that configures CMake to only search for
+libraries in the rootpath of the SDK with
+`CMAKE_FIND_ROOT_PATH <https://cmake.org/cmake/help/v3.31/variable/CMAKE_FIND_ROOT_PATH.html>`_. This is
+something that has to be patched to allow CMake to find libraries in the Conan cache as well:
+
+.. code-block:: cmake
+   :caption: *sdk/sysroots/x86_64-pokysdk-linux/usr/share/cmake/OEToolchainConfig.cmake*
+
+    set( CMAKE_FIND_ROOT_PATH $ENV{OECORE_TARGET_SYSROOT} $ENV{OECORE_NATIVE_SYSROOT} )
+    set( CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER )
+    # COMMENT THIS: set( CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY )
+    # COMMENT THIS: set( CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY )
+    # COMMENT THIS: set( CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY )
+
+You can read more about those variables here:
+
+  - `CMAKE_FIND_ROOT_PATH_MODE_LIBRARY <https://cmake.org/cmake/help/v3.31/variable/CMAKE_FIND_ROOT_PATH_MODE_LIBRARY.html>`_
+  - `CMAKE_FIND_ROOT_PATH_MODE_INCLUDE <https://cmake.org/cmake/help/v3.31/variable/CMAKE_FIND_ROOT_PATH_MODE_INCLUDE.html>`_
+  - `CMAKE_FIND_ROOT_PATH_MODE_PACKAGE <https://cmake.org/cmake/help/v3.31/variable/CMAKE_FIND_ROOT_PATH_MODE_PACKAGE.html>`_
+
+Cross-building Conan packages with the SDK toolchain
+----------------------------------------------------
+
+After setting up your desired SDK, you can start creating Conan packages setting up the environment of the Yocto SDK and running a
+:command:`conan create` command with a suitable profile with the specific architecture of the toolchain.
+
+For example, creating packages for `arch=armv8`:
+
+The profile will be:
+
+.. code-block:: text
+   :caption: *armv8*
+
+    [settings]
+    os_build=Linux
```

Comment:
``os_build`` doesn't exist anymore???

Reply:
Fix by the commit 49d6fc0ab8843f3356d2089bbf44270dd71d0e8e